.pipelines/build-and-push-images-tagged.yml

```diff
@@ -0,0 +1,52 @@
+# Azure DevOps Pipeline building rp images and pushing to int acr
```

Comment:
Newly added pipeline chained after successful E2E pipeline when tag is the trigger. 

Requires adding as new pipeline to the AZP.

Duplicates the original pipeline, however it produces tagged image release. It also creates github release, which requires GitHub service connection.